portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewCreationActions.testcase

```diff
@@ -0,0 +1,526 @@
+				locator1 = "Message#SUCCESS",
+				value1 = "Your request completed successfully.");
+
+			AssertElementNotPresent(
+				key_position = 1,
+				keyName = "Edit",
+				locator1 = "DataSet#FIELDS_TABLE");
+		}
+	}
+
+	@description = "LPS-194351 Confirm that when trying to delete a created action, the deletion can be canceled using the cancel button"
+	@priority = 3
+	test CanCancelDeletionOnCancelButton {
+		task ("And clicks on Create Item Action button") {
+			DataSetAdmin.addActions(
+				actionName = "Edit",
+				actionTab = "Creation Actions",
+				icon = "pencil",
+				readlessAction = "xx-ee",
+				type = "Link",
+				url = "https://gist.github.com/deborasoliveira/40716d0b03d33ff7e636ca5d366c79c5#file-link-action-url-example");
+		}
+
+		task ("And clicks on the Edit option and clicks on delete button") {
+			Click(
+				key_index = 2,
+				key_text = "Actions",
+				locator1 = "DataSet#VERTICAL_ELLIPSIS_N");
+
+			Click(
+				key_menuItem = "Delete",
+				locator1 = "MenuItem#DROPDOWN_MENU_ITEM");
+		}
+
+		task ("And clicking on Cancel button inside the modal") {
+			Button.clickCancel();
+		}
+
+		task ("Then The item is not deleted") {
+			AssertElementNotPresent(
+				key_position = 1,
+				keyName = "Edit",
+				locator1 = "DataSet#FIELDS_TABLE");
+		}
+	}
+
+	@description = "LPS-194351 Confirm that editing an action is canceled when the user clicks the cancel button"
+	@priority = 3
+	test CanCancelEditActionOnCancelButton {
+		task ("And clicks on Create Item Action button") {
+			DataSetAdmin.addActions(
+				actionName = "Edit",
+				actionTab = "Creation Actions",
+				icon = "pencil",
+				readlessAction = "xx-ee",
+				type = "Link",
+				url = "https://gist.github.com/deborasoliveira/40716d0b03d33ff7e636ca5d366c79c5#file-link-action-url-example");
+		}
+
+		task ("And clicks on the Edit option") {
+			Click(
+				key_index = 2,
+				key_text = "Actions",
+				locator1 = "DataSet#VERTICAL_ELLIPSIS_N");
+
+			Click(
+				key_menuItem = "Edit",
+				locator1 = "MenuItem#DROPDOWN_MENU_ITEM");
+		}
+
+		task ("And Edit the action item created, and click on "Cancel" button") {
+			Type(
+				key_unnamedTitle = "",
+				locator1 = "TextInput#DEPOT_UNNAMED_DEFAULT_TITLE",
+				value1 = "Edited label");
+
+			Type(
+				key_fieldName = "Add a value here",
+				locator1 = "DataSet#PRESELECT_VALUE_INPUT",
+				value1 = "");
+
+			Type(
+				key_fieldId = "URL",
+				locator1 = "TextArea#ANY_ID",
+				value1 = "https://gist.github.com/deborasoliveira/40716d0b03d33ff7e636ca5d366c79c5#file-link-action-url-example");
+
+			Button.clickCancel();
+		}
+
+		task ("And it appears on the creation Action tab table with the original label.") {
+			AssertElementPresent(
+				key_position = 1,
+				keyName = "Edit",
+				locator1 = "DataSet#FIELDS_TABLE");
+		}
+	}
+
+	@description = "LPS-194351 Confirm that a created action can be edited and changes are displayed"
+	@priority = 5
+	test CanConfirmChangesInActionCreated {
```

Comment:
Change the name to something more simple: `CanEditCreationAction` is easier to understand